pkg/minikube/cluster/cluster_windows.go

```diff
@@ -41,14 +40,18 @@ func createHypervHost(config MachineConfig) drivers.Driver {
 	return d
 }
 
-func getVMHostIP(host *host.Host) (net.IP, error) {
+func GetVMHostIP(host *host.Host) (net.IP, error) {
```

Comment:
Do you think we could move this up to the main cluster.go file and drop the _windows, _darwin variants? The switch/case should be able to handle this by itself.